src/lib/dev-environment/dev-environment-core.js

```diff
@@ -308,7 +341,7 @@ export async function getApplicationInformation( appId: number, envType: string
 	return appData;
 }
 
-export async function resolveImportPath( slug: string, fileName: string, searchReplace: string, inPlace: boolean ): Promise<SQLImportPaths> {
+export async function resolveImportPath( slug: string, fileName: string, searchReplace: String | String[], inPlace: boolean ): Promise<SQLImportPaths> {
```

Comment:
`string` should be lowercase

Reply:
lowercase causes flow issue here: https://github.com/Automattic/vip/blob/fbb6502fea2304bdd69dfc97623cb41fd2d064be/src/lib/search-and-replace.js#L128

But maybe we should change type there instead of here.

Reply:
Fixed it at the source ...